cognite/powerops/client/shop/api/dayahead_trigger_api.py

```diff
@@ -24,21 +23,21 @@ def __init__(self, client: CogniteClient, data_set: int, cogshop_version: str =
         self.shop_run = SHOPRunAPI(client, data_set, cogshop_version=cogshop_version)
 
     def _create_partial_function_events(
-        self, workflow: DayaheadTrigger, workflow_event_external_id: str, bid_date: str, short_id: str
+        self, workflow: DayaheadTrigger, workflow_event_external_id: str, short_id: str, plants_per_workflow: list[str]
```

Comment:
Could you come up with another name than `short_id`. It seems like it is an unique suffix for all resources created for a the run, so would `suffix_run_id` be a descriptive name?